.github/workflows/build.yml

```diff
@@ -43,54 +48,206 @@ jobs:
           vcpkgArguments: "--triplet=x86-windows-static-md --disable-metrics"
 
       # Install vcpkg dependencies
-      - name: Install & build dependencies
+      - name: ↳ Install & build dependencies
         if: steps.vcpkg_cache.outputs.cache-hit != 'true'
         shell: bash
         run: |
           vcpkg/vcpkg.exe install --triplet=x86-windows-static-md --disable-metrics
 
-          if [ ! -d vcpkg_installed/nwn-lib-d-tools/ ]; then
-            curl -L 'https://github.com/CromFr/nwn-lib-d/releases/download/v1.0.2/nwn-lib-d-tools-windows-x86_64.zip' -o /tmp/nwn-lib-d-tools.zip
-            unzip -d vcpkg_installed/ /tmp/nwn-lib-d-tools.zip
```

Comment:
These nwn-lib-d-tools should be in its own directory. Maybe tools/nwn-lib-d? vcpkg_installed should be only for vcpkg installed packages.

Reply:
those were removed lines, these tools are now installed at C:\nwn-lib-d-tools ;)
https://github.com/nwn2dev/nwnx4/pull/9/files/8071da755fbdc8abbbd0dfa0df4afbd16561cf72#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R72-R86